test-tools/pom.xml

```
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.kie.cloud.tests</groupId>
+  <artifactId>test-tools-parent</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>KIE :: Cloud Integration Test Tools</name>
```

What about project naming like this?
```suggestion
  <name>KIE :: Cloud :: Tools :: Integration Test Tools</name>
```

---

would go for "KIE :: Cloud :: Tools :: Integration Test"

---

Integration Test may be confusing, somebody new could expect that in this module are integration tests for tools

---

right ... did not think about that. Go for Integration Test Tools then :)